docs/deploy-monitoring-opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+---
+id: deploy-monitoring-opentelemetry
+title: OpenTelemetry Monitoring
+sidebar_label: "OpenTelemetry Monitoring"
+---
+
+# Status
+Pulsar emits OpenTelemetry metrics starting from version 3.3.0. OpenTelemetry support is currently **experimental** and
+complements the pre-existing Prometheus metric system, with the goal of eventually replacing it. The metrics it exposes
+are semantically equivalent to the Prometheus metrics.
+
+# Scope
+Pulsar OpenTelemetry metrics are currently available for the broker only. Support for the proxy and function worker is
+planned for a future release.
+
+# OpenTelemetry Configuration
+Pulsar use the auto-configuration [extension](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md)
+of OpenTelemetry to manage the SDK configuration. The extension allows parameter input from environment variables and
+Java system properties. The instructions below rely on environment variables, but can be adapted to use system
+properties too. These variables must be exposed to the Pulsar process via the respective deployment method.
+
+Note that the experimental file based configuration is not currently supported by Pulsar.
```

Comment:
I meant to say it's not _supported_ only. It might very well work, but it's an experimental feature that we haven't validated against our setup.

Reply:
It won't work per the reasons I wrote, and also I believe you need to include another JAR for it.

Reply:
Added hyperlink. We can add a note of support when we implement it. I don't think we need to mention roadmap items in this document, especially since they are not a priority as of now. What do you think?